src/components/app.tsx

```diff
@@ -164,6 +164,10 @@ export class AppComponent extends BaseComponent<IProps> {
   public render() {
     const {appConfig, user, ui, db} = this.stores;
 
+    if (ui.standalone) {
+      return this.renderApp(<AppContentContainerComponent />);
+    }
```

Comment:
This is showing as not being covered by tests -- which seems like a bug since you have `standalone_load_spec` specifically for exercising it.  Is that spec not in the set that is run by CI?

Reply:
I believe I fixed this in https://github.com/concord-consortium/collaborative-learning/pull/2486/commits/d37621f9ab623bdcd0908aef2a9b80d58e82921b - the test needed to be added to the regression test list.  It turns out we only run the one test in the smoke test folder on normal PRs.